### PR TITLE
Add Tilt for running UI and API together

### PR DESCRIPTION
**Pull request by brownbearsoftware**
Login with: JimSnowLdap & secret

Usable CRN: X320741

----

To start run `tilt up` - this starts the docker-compose.yml up with some services not started.

Need to run the api project locally.

Go to http://localhost:10350/ for the Tilt Web UI where you can enable or disable services.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,5 +1,52 @@
...
+        - SYSTEM_CLIENT_SECRET=clientsecret
+        - SESSION_SECRET=app-insecure-default-session
+        - NODE_ENV=development
+        - TOKEN_VERIFICATION_API_URL=http://hmpps-auth:8080/verification
+        - APPROVED_PREMISES_API_URL=http://host.docker.internal:8080
+        - INGRESS_URL=http://localhost:3000
+      ports:
+        - 3000:3000
+        - 9229:9229
+      entrypoint: "node --inspect-brk=0.0.0.0 dist/server.js | bunyan"
```
Comment (edavey) [inline docker-compose.yml:48]: I think this either needs to be commented out or shifted right if we want to run this container also?

Comment (brownbearsoftware) [reply docker-compose.yml]: right you are!  Not sure what happened there, have sorted

**Comment by brownbearsoftware** [comment]
[Add Tilt to UI and API repos so whole system can be run together locally](https://trello.com/c/CtLyRDld/769-add-tilt-to-ui-and-api-repos-so-whole-system-can-be-run-together-locally)

**Comment by pezholio** [comment]
Gonna close this, as it's now in https://github.com/ministryofjustice/hmpps-approved-premises-tools

**Pull request closed without merging**